documentation/homework/week1/todo-app/src/components/todo-item.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
+
+export default class todoItem extends Component {
```

Comment:
React components should always be named with an uppercase first letter. So this component should be called `TodoItem`, and the filename should match: `TodoItem.js`.